samplePDF/samplePDFFDBase.cpp

```diff
@@ -1,37 +1,34 @@
 #include "samplePDFFDBase.h"
 
-// Constructors for erec-binned errors
+#include "OscProbCalcer/OscProbCalcerFactory.h"
+#include "Oscillator/OscillatorFactory.h"
+#include "Constants/OscillatorConstants.h"
 
-samplePDFFDBase::samplePDFFDBase(double pot, std::string mc_version, covarianceXsec* xsec_cov)
+#include<algorithm>
+
+
+samplePDFFDBase::samplePDFFDBase(double pot, std::string mc_version_, covarianceXsec* xsec_cov)
   : samplePDFBase()
```

Comment:
Do we need to pass pot ?
Doesn't it make more sense to store POT in sample config?

Reply:
It's a hangover from previous stuff that should definitely be cleaned...

It even get's void-ed in the base constructor: https://github.com/mach3-software/MaCh3/blob/5fa87ded750b3ba2f040e7eddd9e51e7dcc7350b/samplePDF/samplePDFFDBase.cpp#L10-L14

Reply:
This has now been removed and should be added to any beam-only samples as a member variable